Example-Mac/AppDelegate.swift

```diff
@@ -55,7 +55,28 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             document?.linksBaseURL = linksBaseUrl
         }
         
-        let renderer = CMAttributedStringRenderer(document: document, attributes: CMTextAttributes())!
+        let textAttributes = CMTextAttributes()!;
```

Comment:
Is the force unwrap for CMTextAttributes necessary here? It looks like -[CMTextAttributes init] never returns nil, so putting a `nonnull` annotation on the return type might help avoid this.

Reply:
Also, remove semicolon at end of line

Reply:
Correct. Fixed by assuming non-null in `CMTextAttributes` in commit a5db7ab81f364a6605b601c1a8bf222fa9fb4322